components/popover/__tests__/popover.browser-test.jsx

```diff
@@ -244,6 +249,35 @@ describe('SLDSPopover', function () {
 		});
 	});
 
+	describe('focus has moved to dialog', function () {
+		const triggerClicked = sinon.spy();
+
+		beforeEach(() => {
+			mountNode = createMountNode({ context: this });
+		});
+
+		afterEach(() => {
+			// destroyMountNode({ wrapper, mountNode });
```

Comment:
If you're not using this should it be removed (along with the whole afterEach callback?)